sdk/keyvault/azure-security-keyvault/azure/security/keyvault/certificates/_models.py

```diff
@@ -0,0 +1,609 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from datetime import datetime
+from typing import Any, Dict, Mapping, Optional
+from .._internal import _parse_vault_id
+from .._generated.v7_0 import models
+
+
+class Error(object):
```

Comment:
Would we use this directly? The generated operations I've looked at all raise `KeyVaultErrorException`; we'd either wrap or propagate that.

Reply:
I am not sure how we are exactly going to use this but this is being used [here](https://github.com/Azure/azure-sdk-for-python/pull/5601/files#diff-4acfecdce2f075b1a55c011636405d9cR172)
Do we still want to wrap this into a `KeyVaultErrorException`?

Reply:
Maybe. For now I think it belongs on the pile of things to figure out in implementation.